source/includes/azure/_subnets.md

```diff
@@ -111,6 +111,7 @@ Attributes | &nbsp;
 `addressPrefix`<br/>*string* | The IPv4 CIDR representing the address range for the subnet.
 `id`<br/>*string* | The id associated to the subnet. This is a canonized id from azure which is the form of `/subscriptions/:subscriptionid/resourceGroups/:resourcegroup/providers/Microsoft.Network/virtualNetworks/:networkName/subnets/:subnetName`
 `parentNetworkId` <br/>*string* | The id for the parent network. 
+`parentNetworkName` <br/>*string* | The name for the parent network.
```

Comment:
these attributes all end in periods, but the ones in instances did not, not sure if making them consistent across all files is 'within scope' or if it should be covered in the api docs consistency story

Reply:
I fixed them in instance to have all periods

Reply:
7d86cd5d2eaed27328435a666935632ba6db7c88